### ISSUE TITLE
Preference center background color

### ISSUE THREAD

**Issue by h0shy** [urbanairship/ios-library#320]
# Preliminary Info

### What Airship dependencies are you using?
16.4.0

### What are the versions of any relevant development tools you are using?
Xcode 13.4 / iOS 15.5

# Report

### What unexpected behavior are you seeing?
You cannot completely colorize the preference center background. A narrow area remains in the default color.

### What is the expected behavior?
You can colorize the entire background.

### What are the steps to reproduce the unexpected behavior?

```
let viewController = PreferenceCenterViewController(identifier: id)
viewController.view.backgroundColor = resources.backgroundColor // this is a hack to colorize another rectangle underneath the content.
style.backgroundColor = resources.backgroundColor
```

![Simulator Screen Shot - iPhone 13 - 2022-05-19 at 08 57 39](https://user-images.githubusercontent.com/30653817/169231322-e9c326a9-e6fa-4888-8f2c-35b1f2ada694.png)

**Comment by rlepinski** [comment]
Thanks for the report. We have someone looking at it

**Comment by rlepinski** [comment]
We have a fix internally, hopefully we will get it out next week

**Comment by rlepinski** [comment]
Fixed in 16.8.0
